ext/standard/string.c

```diff
@@ -2751,56 +2753,152 @@ PHP_FUNCTION(quotemeta)
 }
 /* }}} */
 
-/* {{{ proto int ord(string character)
-   Returns ASCII value of character
+/* {{{ proto int ord(string character[, string encoding])
+   Returns code point of character
    Warning: This function is special-cased by zend_compile.c and so is bypassed for constant string argument */
 PHP_FUNCTION(ord)
 {
-	char   *str;
-	size_t str_len;
+  char   *str;
```

Comment:
It seems that whitespace here is done using spaces, and not tabs. Please see CODING_STANDARDS. Alternatively we have the vim rules in the bottom of each file